backend/src/v5/utils/quota.js

```diff
@@ -15,38 +15,50 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 const { createResponseCode, templates } = require('./responseCodes');
+const { getAllUsersInTeamspace, getSubscriptions } = require('../models/teamspaces');
 const DBHandler = require('../handler/db');
 const config = require('./config');
-const { getSubscriptions } = require('../models/teamspaces');
+const { getInvitationsByTeamspace } = require('../models/invitations');
 const { getTotalSize } = require('../models/fileRefs');
 
 const Quota = {};
 
-const calculateQuota = async (teamspace) => {
+Quota.getQuotaInfo = async (teamspace, inMegabytes = false) => {
```

Comment:
Just return bytes, since we return bytes in v4 anyway.


```suggestion
Quota.getQuotaInfo = async (teamspace) => {
```

Reply:
sure, the issue description says megabytes thats why I did it